### PR TITLE
fix(monolith): Bump microgenerator versions to monolith removal

### PR DESCRIPTION
**Pull request by miraleung**
As requested by @bshaffer for the [JWT auth changes](https://github.com/googleapis/gapic-generator-php/pull/309) in the PHP microgenerator.

### CODE REVIEW

**Comment by miraleung** [comment]
Getting this error on a local build, any Bazel-specific insight would be appreciated:

```
Use --sandbox_debug to see verbose messages from the sandbox
google/cloud/compute/v1/compute.proto:28:1: warning: Import google/api/resource.proto is unused.
/usr/local/google/home/miraleung/.cache/bazel/_bazel_miraleung/5f1a5ba2034ce8379afa557b0581ccbf/sandbox/linux-sandbox/531/execroot/com_google_googleapis_discovery/bazel-out/host/bin/external/com_google_protoc_java_resource_names_plugin/gapic_plugin.runfiles/pypi_chevron/chevron/renderer.py:73: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if scope is 0:
Traceback (most recent call last):
  File "/usr/local/google/home/miraleung/.cache/bazel/_bazel_miraleung/5f1a5ba2034ce8379afa557b0581ccbf/sandbox/linux-sandbox/531/execroot/com_google_googleapis_discovery/bazel-out/host/bin/external/com_google_protoc_java_resource_names_plugin/gapic_plugin.runfiles/com_google_protoc_java_resource_names_plugin/plugin/cli/gapic_plugin.py", line 36, in <module>
    from google.protobuf.compiler import plugin_pb2 as plugin
ModuleNotFoundError: No module named 'google.protobuf'
----------------
Note: The failure of target @com_google_protoc_java_resource_names_plugin//:gapic_plugin (with exit code 1) may have been caused by the fact that it is running under Python 3 instead of Python 2. Examine the error to determine if that appears to be the problem. Since this target is built in the host configuration, the only way to change its version is to set --host_force_python=PY2, which affects the entire build.

If this error started occurring in Bazel 0.27 and later, it may be because the Python toolchain now enforces that targets analyzed as PY2 and PY3 run under a Python 2 and Python 3 interpreter, respectively. See https://github.com/bazelbuild/bazel/issues/7899 for more information.
----------------
--resourcename_out: protoc-gen-resourcename: Plugin failed with status code 1.
ERROR: /usr/local/google/home/miraleung/dev/googleapis-discovery/google/cloud/compute/v1/BUILD.bazel:148:26 Building google/cloud/compute/v1/libcompute_java_gapic.jar (1 source jar) failed: (Exit 1): protoc failed: error executing command bazel-out/host/bin/external/com_google_protobuf/protoc --experimental_allow_proto3_optional ... (remaining 12 argument(s) skipped)
```

File: WORKSPACE [hunk 1]
```diff
@@ -403,9 +403,9 @@ gapic_generator_csharp_repositories()
 ##############################################################################
 # Ruby
 ##############################################################################
-_gapic_generator_ruby_version = "e10d40afa96a28036da03bb9b0af17d702715886"
+_gapic_generator_ruby_version = "2b66e7aca8d5d7d4cb7bf436776d7713d264cab8"
```
Comment (vam-google) [inline WORKSPACE:406]: just in case: this is the newer version, rigth?

**Review by vam-google — APPROVED** [review]
LGTM

File: WORKSPACE (hunk 1 above)
Comment (miraleung) [reply WORKSPACE:406]: Yes - this [matches](https://github.com/googleapis/googleapis/blob/master/WORKSPACE#L361) the latest hash that is in googleapis.

**Pull request merged**